internal/functions/coviddata/download-covid-data-total_test.go

```diff
@@ -35,7 +35,10 @@ func (c *ClientMock) Do(req *http.Request) (*http.Response, error) {
             "potvrzene_pripady_dnesni_den_datum": "2021-01-11",
             "provedene_antigenni_testy_celkem": 1037613,
             "provedene_antigenni_testy_vcerejsi_den": 9743,
-            "provedene_antigenni_testy_vcerejsi_den_datum": "2021-01-10"
+            "provedene_antigenni_testy_vcerejsi_den_datum": "2021-01-10",
+			"vykazana_ockovani_celkem": 581542,
```

Comment:
I know, it's just formatting of sth in tests, but... :smile:

Reply:
oh, well... spaces and tab look same in the IDE :facepalm: